stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/MediaAttachmentContent.kt

```diff
@@ -419,6 +422,12 @@ internal fun MediaAttachmentContentItem(
             .build()
     }
 
+    // Prepare the image loader for the media gallery
+    val imageLoader = LocalStreamImageLoader.current
+    LaunchedEffect(imageLoader) {
+        MediaGalleryInjector.install(imageLoader)
+    }
+
```

Comment:
I'm just thinking out loud... this installation happens on every content item composition. Shouldn't it be moved higher in the chain instead? 🤔

Reply:
Considering this composable might be called many times when it's a multiple attachment, maybe installing it in `MediaAttachmentContent`
WDYT?
Given you have touched more in this area than me, feel free to keep it this way or change it if you think it makes more sense.